asset_dashboard/services.py

```diff
@@ -0,0 +1,68 @@
+import json
```

Comment:
Will DRF be able to cleanly distinguish the different tables that we'll need to search? I thought that having a search service will allow us to have flexibility for searching the geo assets. We'll need to search these tables in their GIS database:
- building
- trails
- points of interest
- parking lots
- picnic grove
- signage

The fields we search aren't consistent between those tables, either. So we'll have to come up with a good way to handle these different cases. For instance, we'll need to search for `fpd_uid` and `name` in the building table, but those don't exist in the trails. For the trails, Garret said we'll need to search by subsystem of the trails. For points of interest, we'll need to filter where `poi_info_id == poin_info_id_group`. 

And re, the services file, I was trying to move to a more organized approach where we could store the different business logic outside of the views, and access what we need in the views, so that we could make the code more organized and maintainable. So ideally the services could be any type of service that we would want to use or reuse in a view. It keeps the view code cleaner by encapsulating that logic that shouldn't live in a view (imo, would be curious to hear other perspectives!)

Reply:
I think there's a clean way to be had with DRF, yes. I spiked one idea here: https://github.com/fpdcc/ccfp-asset-dashboard/pull/101

Re: services, it feels a bit code smelly to move code that only one view uses into a separate file. If we write a Django form or an object with a similar API and use a form view, I would expect to find this sort of business logic on the view, either in the `get_context_method` or `form_valid` method.

But let's come back to this discussion once we settle on whether to use DRF or the form pattern.